src/decorators/attributes.js

```diff
@@ -36,7 +35,8 @@ function defaultValueForDescriptor (descriptor, DataType) {
   if (typeof descriptor.initializer === 'function') {
     const value = descriptor.initializer()
     if (DataType.name === 'Date') {
-      return moment(value).toDate()
+      if (value instanceof Date) return value
+      return new Date(Date.parse(value))
```

Comment:
Same goes for here.

Reply:
it mostly works as-is, but converting between moment and native dates, we actually lose millisecond precision. i haven't tracked that down yet (and i'm not sure it's important), but i can do a `value._isAMomentObject` here as well to just pass it on thru untouched.

Reply:
That might be a good defensive measure.